go.mod

```diff
@@ -41,6 +41,7 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
+	github.com/codeready-toolchain/toolchain-common => github.com/ranakan19/toolchain-common v0.0.0-20210518183450-d7c417a42bd9
```

Comment:
You don't need it, do you?

Reply:
no. I was wondering if it would be of any help to keep it nevertheless, like in sync everywhere, but i guess just setting blockOwnerDeletion without other details of owner reference doesn't help anyone. 
I'll go ahead and remove this.